app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -17,6 +17,7 @@ import org.mozilla.tv.firefox.telemetry.TelemetryIntegration
 import org.mozilla.tv.firefox.utils.UrlUtils
 
 data class ValidatedIntentData(val url: String, val source: Session.Source)
+private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
```

Comment:
nit: Maybe we should change the value to `qaActiveExperiments`? That will make it clear to QA (and pesky users!) that they should not have many expectations when using this functionality. I asked Daniela to make this change too.

Reply:
I'll update the wiki page too.